src/components/display/Navbar.tsx

```diff
@@ -19,7 +19,18 @@ const Navbar = () => (
       ))}
     </div>
     <div className="navbar-end" />
-    <Avatar firstName="John"  lastName="Doe" username="JonnyDoe" />
+    
+    <div className="dropdown dropdown-end">
+      <div tabIndex={0} className="btn btn-ghost rounded-btn flex-nowrap p-3">
```

Comment:
Do we need this div and ul be focusable? (I'm not in front of my computer).

Reply:
Mmm maybe  I haven't considered that when adding elements. This is also partially implemented, it requires some polishing